tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -81,19 +81,32 @@ jobs:
 
 - template: templates/win-ci.yml
   parameters:
-    ort_build_pool_name: 'onnxruntime-gpu-winbuild'
+    ort_build_pool_name: 'onnxruntime-gpu-tensorrt8-winbuild'
```

Comment:
You may change line 83 unchanged.  onnxruntime-gpu-winbuild all the things onnxruntime-gpu-tensorrt8-winbuild has, and its GPU driver is newer. 

The GPU driver on onnxruntime-gpu-tensorrt8-winbuild is too old to support CUDA 11.4.

Reply:
Okay, so onnxruntime-gpu-winbuild also so TRT8 libraries now, right?

Reply:
The pipeline failed on some unit tests when switching back to onnxruntime-gpu-winbuild.
Is this expected?